### DISCUSSION TITLE
Limit number of connection and logs

### DISCUSSION THREAD

**Discussion by sirreaver** [any1/wayvnc#290] [Q&A]
Hello and thanks for you work.

Is it possible to limit number of connection? Like x11vnc has  -shared and -nevershared parameters. Actually not need for precise number (but it would be nice) But at least one or more.

And also, are there some logs? Or how I can get them or is the redirect of wayvnc command enough (I actually got empty file)?

Thank you for answers

**Comment by Halfdann** [comment]
By any chance is this possible handle on your own by utilizing otpions/events/commands of connection:

CLIENT-LIST
The client-list command retrieves a list of all VNC clients currently connected to wayvnc.

CLIENT-CONNECTED
The client-connected event is sent when a new VNC client connects to wayvnc.

And if it exceeds your "rules" then to force/close/kick the client(s) ids which are above your line.

Source:
[https://www.mankier.com/1/wayvnc#Options](url)
